karma-cov.conf.js

```diff
@@ -41,13 +41,11 @@ module.exports = function (config) {
       {type: 'text'}
     ]
   };
-  karma_config.webpack.module.preLoaders = [
-    {
-      test: /\.js$/,
-      include: path.resolve('src/'),
-      loader: 'istanbul-instrumenter'
-    }
-  ];
+  karma_config.webpack.module.rules.unshift({
```

Comment:
what is the difference between preloaders vs unshift?

Reply:
looks like API change.

Reply:
`unshift` just adds to the beginning of an array.  There are no longer preloaders in webpack 3; you just need to order your loaders correctly.  This places the loader first.